gax/src/streamingRetryRequest.ts

```diff
@@ -22,6 +22,7 @@ const DEFAULTS = {
     Max # of retries
   */
   maxRetries: 2,
+  request: require('teeny-request'),
```

Comment:
If you want to keep the `require` statement conditional, I could revert to doing this around line 44 in `if (opts.request === undefined)`